leaflet-rotatedmarker/resources/cljsjs/leaflet-rotatedmarker/common/leaflet-rotatedmarker.ext.js

```diff
@@ -0,0 +1,5 @@
+var rotatedMarker {
```

Comment:
```
./leaflet-rotatedmarker/resources/cljsjs/leaflet-rotatedmarker/common/leaflet-rotatedmarker.ext.js:1: ERROR - Parse error. Semi-colon expected
var rotatedMarker {
```

Reply:
Still broken, the error message is in fact confusing, the problem is missing `=`. Please test packages locally in future, with advanced compilation so Closure will try to read the extern.

Reply:
Could you expand on the instructions on the Wiki by giving the complete command needed to do this local testing? I'm sure they are understandable by someone who's used boot before though. https://github.com/cljsjs/packages/wiki/Creating-Packages

Reply:
Complete commend to install the package to local Maven repository is mentioned there:

```
boot package install target
```

(target part is optional)

Then just add the dependency to project where you want to use the lib.

Reply:
Sorry to bother you again but I've tested locally with `boot package install target` and the library seems to work when I use it in another project. It was the part about advanced compilation that I didn't get from the Wiki.

Reply:
The advanced part just means that you should use `:optimization :advanced` in the project where you use the library so that the extern gets used. As the extern file is still invalid JavaScript, if you don't see the problem currently, the extern file is not used.

Reply:
Anyway, as quick fix (if you are not able to test this with advanced compilation), change this line:

```
var rotatedMarker {
=>
var rotatedMarker = {
```

Reply:
Oh, for some reason I thought I had already pushed that change. But it should be up now.